charm/ksql/actions.yaml

```diff
@@ -0,0 +1,8 @@
+create-ksql:
+  description: Runs the ksql scripts to create streams/tables.
+  params:
+    entries_yaml:
```

Comment:
Juju won't allow parameter names with underscores... ran into the same issue w/cert-manager-charm. Had to use dashes.